src/shared/components/case-editor/services/wizard-page-field-to-case-field.mapper.ts

```diff
@@ -64,6 +66,33 @@ export class WizardPageFieldToCaseFieldMapper {
     }
   }
 
+  private fixShowConditionPath(caseField: CaseField, pathPrefix: string) {
+    if (caseField.show_condition && pathPrefix.length > 0 && !caseField.show_condition.startsWith(pathPrefix)) {
+      caseField.show_condition = pathPrefix.length > 0 ? pathPrefix + '.' + caseField.show_condition : caseField.show_condition;
+    }
+
+    let childrenCaseFields = [];
+    if (this.isCollection(caseField)) {
```

Comment:
These 5 lines can be converted into a function to return child complex fields. I see them repeated in hideParentIfAllChildrenHidden.

Reply:
👍